src/js/app.js

```diff
@@ -0,0 +1,39 @@
+import { timeDifference, timeDifferenceFuture } from './utils'
+
+export default function init() {
+  document.getElementById('ago').textContent = `(${timeDifference(
```

Comment:
Template literal with a function is expensive, why not assign it to a variable first?

https://jsperf.com/es6-string-literals-vs-string-concatenation